src/components/Footer.js

```diff
@@ -71,7 +82,24 @@ const Footer = () => (
             <Medium />
           </a>
         </SocialLinks>
-        <Copyrights>Copyright {new Date().getFullYear()} © Astrocoders Desenvolvimento LTDA. CNPJ 24.896.711/0001-54. Avenida Afonso Vergueiro, 2250, Sala 2, Sorocaba, São Paulo.</Copyrights>
+        <Copyrights>
+          Copyright {new Date().getFullYear()} © Astrocoders Desenvolvimento LTDA. CNPJ 24.896.711/0001-54. Avenida
+          Afonso Vergueiro, 2250, Sala 2, Sorocaba, São Paulo.
+          <CopyrightIcons>
+            Icons made by{' '}
+            <a href="https://www.flaticon.com/authors/eucalyp" title="Eucalyp">
```

Comment:
this should be a styled component also no?

Reply:
not really I think 🤔